server/test/unit/compute/test_diffexp_cxg.py

```diff
@@ -19,8 +19,8 @@ class DiffExpTest(unittest.TestCase):
 
     def load_dataset(self, path, extra_server_config={}, extra_dataset_config={}):
         config = app_config(path, extra_server_config=extra_server_config, extra_dataset_config=extra_dataset_config)
-        loader = MatrixDataLoader(path)
-        adaptor = loader.open(config)
+        loader = MatrixDataLoader(path, config)
```

Comment:
It might be useful to add a test where the dataset_config and matrix_data_type are passed in, and even one with an incorrect data type

Reply:
Yes, I can write a test case for that.  I'll put it in test/unit/data_common/test_matrix_loader.py